.github/actions/notebooks/action.yml

```diff
@@ -0,0 +1,18 @@
+name: Labelbox Example Notebook Setup
```

Comment:
Are you sure you don't want to use the same one here https://github.com/Labelbox/labelbox-python/blob/develop/.github/actions/python-package-shared-setup/action.yml as it looks very similar?

Reply:
Done